dc.local.yml

```diff
@@ -0,0 +1,15 @@
+# This file contains configuration defaults for base/local environments
```

Comment:
I'm still unclear on why we need this file.. You should be able to run everything locally with the base `docker-compose` configuration.

Reply:
Yeah I was testing locally with the containers name without the docker URL included, I found it handy to include this file but you are right that it's not totally needed.

Reply:
We should remove it if its not necessary, cleans up the repo and less to maintain in the future